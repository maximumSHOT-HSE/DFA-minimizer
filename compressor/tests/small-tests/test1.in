7 21 3
1 7 0
1 5 1
1 3 2
2 6 0
2 5 1
2 2 2
3 7 0
3 7 1
3 1 2
4 2 0
4 6 1
4 3 2
5 7 0
5 5 1
5 1 2
6 5 0
6 2 1
6 3 2
7 4 0
7 1 1
7 5 2
6
1
7 
