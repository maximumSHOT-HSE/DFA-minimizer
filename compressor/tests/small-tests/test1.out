7 21 3
7 1 1
3 1 2
5 1 2
4 2 0
6 2 1
2 2 2
1 3 2
4 3 2
6 3 2
7 4 0
6 5 0
1 5 1
2 5 1
5 5 1
7 5 2
2 6 0
4 6 1
1 7 0
3 7 0
5 7 0
3 7 1
6
1
7 
